db/db.sql

```diff
@@ -15,7 +15,8 @@ CREATE TABLE `jobs` (
   `updatedAt` datetime not null,
   `request` varchar(4096) not null default 'unknown',
   `isAsync` boolean,
-  `numInputGranules` integer not null default 0);
+  `numInputGranules` integer not null default 0,
+  `shapeFileUrl` varchar(4096) not null);
```

Comment:
Does it make sense to pre-compute the shapefile md5 and just store that instead of the url?

Reply:
You're right - I'll get that changed.

Reply:
This is now done.